tests/testthat/test-checks_df.R

```diff
@@ -1,3 +1,14 @@
+examples_path <- system.file("example_packages", package = "checked")
+names <- c("DALEXtra_new", "rd2markdown_new", "exampleGood_new", "exampleOkay_new", "exampleBad_new")
+path <- path_broken <- c(
+  test_path("testing_pkgs", "DALEXtra"),
+  test_path("testing_pkgs", "rd2markdown"),
+  file.path(examples_path, "exampleGood"),
+  file.path(examples_path, "exampleOkay"),
+  file.path(examples_path, "exampleBad")
+)
```

Comment:
Unclear why we define both `path` and `path_broken` here, and also suggest not defining the variable `names` (as it is ambiguous out-of-context with `base::names()`).

```suggestion
pkg_paths <- c(
  "DALEXtra_new" = test_path("testing_pkgs", "DALEXtra"),
  "rd2markdown_new" = test_path("testing_pkgs", "rd2markdown"),
  "exampleGood_new" = file.path(examples_path, "exampleGood"),
  "exampleOkay_new" = file.path(examples_path, "exampleOkay"),
  "exampleBad_new" = file.path(examples_path, "exampleBad")
)
```

Reply:
They are used later in tests but moved closer to the particular part it is used.